Services/Pages/PageService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.Pages;
+
+namespace Andculture.Sitefinity.Testing.Services.Pages
+{
+    public class PageService : ODataService<Page>
+    {
+        public PageService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system-live/pages";
```

Comment:
Suggestion - Consider moving the path to a configuration store. That way we can quickly update it if it changes over time.